wandb/sdk/wandb_verify.py

```diff
@@ -0,0 +1,493 @@
+"""
+Utilities for wandb verify
+"""
+from __future__ import print_function
+
+import getpass
+import os
+import time
+
+import click
+from gql import gql  # type: ignore
+from pkg_resources import parse_version  # type: ignore
+import requests
+import wandb
+
+
+if wandb.TYPE_CHECKING:  # type: ignore
+    from typing import (
+        Any,
+        Dict,
+        List,
+        Optional,
+        Tuple,
+        Union,
+    )
+
+    from wandb.sdk.wandb_artifacts import Artifact
```

Comment:
use a relative import for things in sdk (once moved it will be)
```
from ..wandb_artifacts import Artifact
```

This import only works because it is under type checking, otherwise it would get the wrong sdk if py27

Reply:
These imports are only used for type checking. For some reason wandb.Artifact is not a valid type for type checking, this was my work around.